src/main/java/employee/controller/EmployeeController.java

```diff
@@ -20,4 +22,15 @@
         Employee newEmployee = employeeRepository.save(employee);
         return new ResponseEntity<>(newEmployee, HttpStatus.CREATED);
     }
+
+    @ResponseBody
+    @RequestMapping(value = "/employees/{id}", method = RequestMethod.GET, produces = "application/json;charset=UTF-8")
+    public ResponseEntity<Employee> getEmployee(@PathVariable("id") long employeeId){
```

Comment:
Even though it is correct that the ID is a number it will cause a failure and leak error messages to the client if you make a request for an employee with a textual ID like `/employees/whatever` so it is best to have the `@PathVariable` as a String and validate it before parsing it into a `Long` and yes a `Long` not `long` as the method `findById()` accepts an Object and not a primitive.

You can use a regular expression to test if the `employeeId` String is a valid digit before converting it into a `Long` and continuing.